pkg/server/workspaces/remove.go

```diff
@@ -70,3 +70,35 @@ func (s *WorkspaceService) RemoveWorkspace(workspaceId string) error {
 	log.Infof("Workspace %s destroyed", workspace.Id)
 	return nil
 }
+
+func (s *WorkspaceService) ForceRemoveWorkspace(workspaceId string) error {
+	// This version of RemoveWorkspace ignores provider errors and continues with the deletion
```

Comment:
Please move this comment above the method and change it to 
```
ForceRemoveWorkspace ignores provider errors and makes sure the workspace is removed from storage.
```

Also, remove the extra newline, there shouldn't be a newline at the start of the function.